macros/external/create_external_table.sql

```diff
@@ -93,6 +93,37 @@
     file_format = {{external.file_format}}
 {% endmacro %}
 
+{% macro sqlserver__create_external_table(source_node) %}
+
+    {%- set columns = source_node.columns.values() -%}
+    {%- set external = source_node.external -%}
+    {%- set partitions = external.partitions -%}
```

Comment:
Do SQLServer / Synapse have the ability to define external tables with partitions?

Reply:
nope! I should probably drop this section